include/openssl/core_names.h

```diff
@@ -349,6 +349,8 @@ extern "C" {
 #define OSSL_PKEY_PARAM_FFC_DIGEST       OSSL_PKEY_PARAM_DIGEST
 #define OSSL_PKEY_PARAM_FFC_DIGEST_PROPS OSSL_PKEY_PARAM_PROPERTIES
 
+#define OSSL_PKEY_PARAM_EC_ENCODING      "ec-param-encoding" /* utf8_string */
```

Comment:
"encoding" should be sufficient.  The context is EC anyway, so the redundancy is not necessary, and is deterring when used as `openssl` pkey options.